API-Samples/Java/app/src/main/java/videoindexersamples/authentication/AccessTokenRequest.java

```diff
@@ -0,0 +1,22 @@
+package videoindexersamples.authentication;
+
+
+/**
+ * The request model for getting Video Indexer Access Token
+ */
+public class AccessTokenRequest {
+    public String videoId;
+    public String projectId;
+    public ArmAccessTokenPermission permissionType;
+    public ArmAccessTokenScope scope;
+
+    public AccessTokenRequest(String videoId, String projectId, ArmAccessTokenPermission permission, ArmAccessTokenScope scope) {
+        this.videoId = videoId;
+        this.projectId = projectId;

```

Comment:
I think that you either need project id or videoId.

Reply:
I see

Reply:
but I cant split it to 2 c'tor